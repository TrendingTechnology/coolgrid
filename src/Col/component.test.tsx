import { shallow } from 'enzyme'
import Component from '.'

describe(Component.displayName, () => {
  describe(TEST_COMPONENT, () => {
    it('component renders correctly', async () => {
      const tree = shallow(<Component />)
      expect(tree).toMatchSnapshot()
    })
  })
})
