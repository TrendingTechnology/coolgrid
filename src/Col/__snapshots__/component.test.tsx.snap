// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mosquito-ui/grid/Col component renders correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
